src/main/scala/beam/router/skim/CsvSkimReader.scala

```diff
@@ -38,15 +36,15 @@ class CsvSkimReader(
       skimsTryMap match {
         case Success(skimMap) => skimMap
         case Failure(ex) =>
-          logger.warn(s"Could not load warmStart skim from '$aggregatedSkimsFilePath': ${ex.getMessage}")
+          logger.warn(s"Could not load warmStart skim from '$aggregatedSkimsFilePath'. $ex")
           Map.empty[AbstractSkimmerKey, AbstractSkimmerInternal]
       }
     }
   }
 
   def readSkims(reader: BufferedReader): Map[AbstractSkimmerKey, AbstractSkimmerInternal] = {
     tryReadSkims(reader).recover { case ex: Throwable =>
-      logger.warn(s"Could not load warmStart skim from '$aggregatedSkimsFilePath': ${ex.getMessage}")
+      logger.warn(s"Could not read warmStart skim from '$aggregatedSkimsFilePath'. $ex")
```

Comment:
We could use the logger's feature to print the whole stack trace if we provide the exception as a parameter to `warn` method:
`, ex)`